docs/admin/about-docs.md

```diff
@@ -59,3 +59,24 @@ cd docs
 make html
 ```
 
+## Documentation Markup Keys
```

Comment:
```suggestion
## FOLIO Apps Documentation Markup Keys
```

Reply:
I think we want to scope this to the Apps documentation. The formatting might be useful in other situations.